jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/RadioButtonPage.java

```diff
@@ -0,0 +1,35 @@
+package io.github.com.pages.inputs;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+
+import java.util.List;
+
+public class RadioButtonPage extends WebPage {
+    @UI("//h2[text()='Simple radio']/following-sibling::fieldset[1]/div/label")
+    public static List<UIElement> simpleRadioButtons;
+
+    @UI("//h2[text()='Simple radio']/following-sibling::fieldset[1]/div/label/span[1]")
```

Comment:
you can use simpleRadioButtons variable for this
it can be deleted

Reply:
Mui-checked is contained in span, so we need both for text and classes
![image](https://user-images.githubusercontent.com/30774377/117123746-172b0500-ada0-11eb-8d2b-8c4d900137e9.png)